data/json/recipes/other/tool.json

```diff
@@ -55,6 +55,36 @@
     "tools": [ [ [ "swage", -1 ] ] ],
     "components": [ [ [ "scrap", 2 ] ] ]
   },
+  {
+    "result": "shears",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "3 h",
+    "autolearn": true,
+    "reversible": true,
+    "using": [ [ "forging_standard", 4 ] ],
+    "qualities": [ { "id": "ANVIL", "level": 3 }, { "id": "HAMMER", "level": 3 }, { "id": "CHISEL", "level": 3 } ],
+    "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "mold_plastic", -1 ] ] ],
+    "components": [ [ [ "plastic_chunk", 1 ] ], [ [ "scrap", 3 ] ] ]
```

Comment:
One lump of steel (which is 1 kg) would better represent the item's current weight than three pieces of scrap metal (150 g). Same for recipe below.